src/queryParameters/extractCreatedByParameters.ts

```diff
@@ -0,0 +1,42 @@
+import { ajv } from '../ajv';
+import { InputValidationError } from '../errors';
+import type { JSONSchemaType } from 'ajv';
+import type { Request } from 'express';
+
+interface CreatedByParameters {
+	createdBy: number | undefined;
+}
+
+interface CreatedByParametersQuery {
```

Comment:
As of this commit, this and the above `CreatedByParameters` are the exact same interface. Would anything bad happen by removing one of them, e.g. this one, and re-using the one interface below?

Reply:
good call!